go/database/mpt/io/interrupt.go

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) 2024 Fantom Foundation
+//
+// Use of this software is governed by the Business Source License included
+// in the LICENSE file and at fantom.foundation/bsl11.
+//
+// Change Date: 2028-4-16
+//
+// On the date above, in accordance with the Business Source License, use of
+// this software will be governed by the GNU Lesser General Public License v3.
+
+package io
+
+import (
+	"context"
+	"errors"
+	"log"
+	"os"
+	"os/signal"
+	"syscall"
+)
+
+var errCanceled = errors.New("export was interrupted")
```

Comment:
Consider using a `ConstError` to make this error immutable. An example is [here](https://github.com/Fantom-foundation/Carmen/blob/cb0032b7724512a252766f973bddcacdc9752c54/go/carmen/database.go#L22).